scripts/verdaccio.yaml

```diff
@@ -2,226 +2,25 @@ storage: .verdaccio-cache
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/types':
-    access: $all
-    publish: $all
-    proxy: npmjs
-
-  # storybook's packages never hosted in this monorepo
-  '@storybook/csf':
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/global':
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/design-system':
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/preset-ie11':
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/preset-scss':
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/bench':
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/addon-coverage':
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/addon-bench':
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/addon-styling':
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/addon-svelte-csf':
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/create-svelte-with-args':
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/react-docgen-typescript-plugin':
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/testing-library':
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/test-runner':
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/expect':
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/docs-mdx':
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/semver':
-    # TODO: remove this when new versions of all dependents have been published and updated. It is not used anymore but still referenced in the latest packages
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/icons':
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/ember-cli-storybook':
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/addon-webpack5-compiler-swc':
-    access: $all
-    publish: $all
-    proxy: npmjs
-  '@storybook/addon-webpack5-compiler-babel':
-    access: $all
-    publish: $all
-    proxy: npmjs
-
-  # storybook packages are NOT proxied to global registry
-  # allowing us to republish any version during tests
-  '@storybook/*':
-    access: $all
-    publish: $all
-
-  'sb':
-    access: $all
-    publish: $all
-
-  'create-storybook':
-    access: $all
-    publish: $all
-
-  'storybook':
-    access: $all
-    publish: $all
-
-  'storybook-addon-pseudo-states':
-    access: $all
-    publish: $all
-
-  'eslint-plugin-storybook':
-    access: $all
-    publish: $all
-
   '@*/*':
     access: $all
     publish: $all
-    proxy: npmjs
+    unpublish: $all
 
   '**':
     access: $all
     publish: $all
-    proxy: npmjs
+    unpublish: $all
+
+log:
+  type: stdout
+  format: pretty
+  level: debug
 
-logs:
-  - { type: stdout, format: pretty, level: warn }
+publish:
+  # no npm connection needed
```

Comment:
Either link to the docs, or no comment at all. I think `allow_offline: true` is pretty self-explanatory.

Docs are here:
https://verdaccio.org/docs/configuration/#offline-publish